app/lib/screens/HomeScreens/chat/ChatScreen.dart

```diff
@@ -262,7 +266,25 @@ class _ChatScreenState extends State<ChatScreen>
           body: Obx(
             () => SafeArea(
               bottom: false,
-              child: buildBody(context),
+              child: networkController.connectionType.value == '0'
+                  ? Center(
```

Comment:
Is this the same widget used everywhere? Maybe define it one place and calling it wherever needed? :)